File: scratch/find_orbit.js
const fs = require('fs');

const content = fs.readFileSync('styles/style.css', 'utf8');
const lines = content.split('\n');

lines.forEach((line, i) => {
    const lineNum = i + 1;
    if (line.includes('orbit-card') || line.includes('orbit-img') || line.includes('orbit-ring')) {
        console.log(`Line ${lineNum}: ${line.trim()}`);
    }
});

File: scratch/find_orbit.py
import re

with open("styles/style.css", "r", encoding="utf-8") as f:
    lines = f.readlines()

for i, line in enumerate(lines, 1):
    if "orbit-card" in line or "orbit-img" in line or "orbit-ring" in line:
        print(f"Line {i}: {line.strip()}")
